recipes/jupyter-widget-pivot-table/meta.yaml

```diff
@@ -1,13 +1,13 @@
-{% set version = "0.1.3" %}
-{% set md5 = "7eebff0d6da4fed51cfb143c4dcd0011" %}
+{% set version = "0.1.3a0" %}
+{% set md5 = "ec10c105aabac9101b109475e26faaef" %}
```

Comment:
Would it be possible to [sha256?](https://conda-forge.org/docs/meta.html#populating-the-hash-field)

Reply:
Sure, it's done !